app/Chatkit Quickstart/SenderMessageTableViewCell.swift

```diff
@@ -2,43 +2,53 @@ import UIKit
 
 class SenderMessageTableViewCell: UITableViewCell {
     
-    @IBOutlet weak var lblName: UILabel!
-    @IBOutlet weak var imgBackground: UIImageView!
-    @IBOutlet weak var lblMessage: UILabel!
-    @IBOutlet weak var imgAvatar: UIImageView!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var backgroundImageView: UIImageView!
+    @IBOutlet weak var messageLabel: UILabel!
+    @IBOutlet weak var avatarImageView: UIImageView!
     
     override func awakeFromNib() {
         super.awakeFromNib()
-        // Initialization code
-        lblName.textAlignment = .right
-        lblMessage.textAlignment = .right
         
-        imgBackground.backgroundColor = UIColor(red: 0.19, green: 0.05, blue: 0.31, alpha: 1.0)
-        imgBackground.layer.cornerRadius = 4.0
-        imgBackground.clipsToBounds = true
+        nameLabel.textAlignment = .right
         
-        lblMessage.textColor = UIColor.white
+        messageLabel.textAlignment = .right
+        messageLabel.textColor = UIColor.white
+        
+        backgroundImageView.backgroundColor = Self.defaultBackgroundColor
+        backgroundImageView.layer.cornerRadius = 4.0
+        backgroundImageView.clipsToBounds = true
     }
     
-    func setBg(color: UIColor) {
-        imgBackground.backgroundColor = color
-    }
 }
 
 extension SenderMessageTableViewCell: MessageTableViewCell {
-    func configure(senderName: String, text: String) {
-        lblName.text = senderName
-        lblMessage.text = text
+
+    static var defaultBackgroundColor: UIColor {
+        return UIColor(red: 0.19, green: 0.05, blue: 0.31, alpha: 1.0)
     }
     
-    func setImage(ImageURL: String) {
-        URLSession.shared.dataTask( with: NSURL(string:ImageURL)! as URL, completionHandler: {
-            (data, response, error) -> Void in
-            DispatchQueue.main.async {
+    func configure(senderName: String,
+                   senderAvatarUrl: String?,
+                   text: String,
+                   backgroundColor: UIColor) {
+        
+        nameLabel.text = senderName
+        messageLabel.text = text
+        backgroundImageView.backgroundColor = backgroundColor
+        avatarImageView.image = UIImage(named: "AvatarPlaceholder")
+        
+        if let senderAvatarUrlString = senderAvatarUrl,
+            let senderAvatarUrl = URL(string: senderAvatarUrlString) {
+    
+            URLSession.shared.dataTask(with: senderAvatarUrl) { (data, _, _) in
```

Comment:
One could argue that it is not the view responsibility to download an image. However, I do agree that it is convenient to do it here.

Reply:
I agree but I guess we're trying to keep this as simple as possible so I don't think its worth splitting out for the puposes of this quick start.

Reply:
I wanted to move it downwards as well, but I was cautioned against putting a `Drawable` property in the view type.